src/components/CommentsList.jsx

```diff
@@ -1,29 +1,32 @@
 import React from "react";
 import CommentCard from "./CommentCard";
 import { getCommentsByArticleId } from "../utils/utils";
-import {useParams} from 'react-router'
+import { useParams } from "react-router";
+import CommentPostForm from "./CommentPostForm";
 
 const CommentsList = () => {
   const [comments, setComments] = React.useState([]);
-  const {article_id} = useParams()
+  const { article_id } = useParams();
 
   React.useEffect(() => {
-    getCommentsByArticleId(article_id).then((data)=>{
-        setComments(data)
-    })
+    getCommentsByArticleId(article_id).then((data) => {
+      setComments(data);
+    });
   }, []);
 
   return (
-    <div className="comments-list">{comments.map((comment)=>(
-    <CommentCard
-    key={comment.comment_id}
-    body={comment.body}
-    author={comment.author}
-    created_at={comment.created_at}
-    votes={comment.votes}
-    />
-  ))}
-  </div>)
-
-}
+    <div className="comments-list">
```

Comment:
Given you have reffered to this as a list maybe it should be in a. < ul> or < ol> tag and the cards all be < li>